### DISCUSSION TITLE
CLI documentation outdated?

### DISCUSSION THREAD

**Discussion by karashiiro** [serverless/serverless#10720] [Q&A] [answered]
Hi all! I came back to Serverless recently (after one of my old functions became outdated) and found that the CLI had changed. Trying to use the CLI as described in the current documentation results in the following error:
```
$ serverless deploy
Serverless Components CLI is no longer bundled with Serverless Framework CLI

To run it, ensure it's installed:
npm install -g @serverless/components

Then run:
components <command> <options>
```

The error message here is clear, but the documentation (both [here](https://www.serverless.com/framework/docs/getting-started) and in the repo README) doesn't reflect this change - is this just a case of outdated docs, or was I doing something funny?

Thanks!

OS: Windows 10
Node version: v16.13.0

**Comment by mnapoli** [comment, marked as answer]
Sorry about the confusion. Serverless Components were a separate project (https://github.com/serverless/components) that was embedded in the `serverless` CLI.

These kinds of projects are now outdated. We put the "Components" CLI in a separate package so that we could:

- remove them from the main `serverless` CLI to make it faster, lighter, and avoid NPM installation warnings
- while still providing the ability to deploy existing Components projects via the external package now

If you use the `serverless` CLI, then you can deploy Serverless Framework projects (which follow the documentation and README that you referenced).

Hopefully that makes more sense.

**Reply by karashiiro** [reply]
Ah, that explains it, thanks for clearing things up 👍
